flow/record/fieldtypes/__init__.py

```diff
@@ -542,23 +542,64 @@ def _unpack(cls, data):
         return data
 
 
+def _is_posixlike_path(path: Any):
+    return isinstance(path, pathlib.PurePath) and path._flavour.sep != "\\" and path._flavour.altsep != "\\"
```

Comment:
```suggestion
    return isinstance(path, pathlib.PurePath) and "\\" not in (path._flavour.sep, path._flavour.altsep)
```

Reply:
Note that this might be slower

Reply:
It will probably be a fraction slower due to the construction of a tuple. Will this be an issue? the `path` field type is already relatively heavy.

Reply:
See my reply to the other thread.